src/utils/ProtocolMetrics.ts

```diff
@@ -105,22 +107,35 @@ function updateTreasuryAssets(transaction: Transaction): string[] {
     let bctERC20 = ERC20.bind(Address.fromString(constants.BCT_ERC20_CONTRACT))
     let treasuryBCT = loadOrCreateTreasuryAsset(transaction.timestamp, constants.BCT_ERC20_CONTRACT)
 
+    // MCO2
+    let mco2ERC20 = ERC20.bind(Address.fromString(constants.MCO2_ERC20_CONTRACT))
+    let treasuryMCO2 = loadOrCreateTreasuryAsset(transaction.timestamp, constants.MCO2_ERC20_CONTRACT)
+
     // Treasury token balance
     treasuryBCT.tokenBalance = toDecimal(bctERC20.balanceOf(treasuryAddress), 18)
-    // Reserve asset so carbon and RFV = token balance
+    treasuryMCO2.tokenBalance = toDecimal(mco2ERC20.balanceOf(treasuryAddress))
```

Comment:
Since the MCO2 contract wasn't deployed at the start of the subgraph syncing, we need to wrap this in an `if` to only update after MCO2 was live on Polygon. Using the bond start block worked locally for me. While indexing the subgraph throws an error without this statement.